packages/blocks/src/api/raw-handling/test/paste-handler.js

```diff
@@ -97,33 +106,51 @@ describe( 'pasteHandler', () => {
 		expect( result.isValid ).toBeTruthy();
 	} );
 
-	it( 'can handle a google docs table with colspan', () => {
+	it( 'can handle a table with thead, tbody and tfoot using rowspan', () => {
 		const [ result ] = pasteHandler( {
-			HTML: googleDocsTableWithColspan,
+			HTML: tableWithHeaderFooterAndBodyUsingRowspan,
 			tagName: 'p',
 			preserveWhiteSpace: false,
 		} );
 
 		expect( console ).toHaveLogged();
 
 		expect( result.attributes ).toEqual( {
```

Comment:
Minor: why not check the serialised block like elsewhere in this file?

Reply:
Do you mean that the following should be checked with strings containing comments?

```html
<!-- wp:table -->
<figure class="wp-block-table"><table>...</table></figure>
<!-- /wp:table -->
```

Reply:
Yes. You could use `.expect().toMatchInlineSnapshot()` I think.

Reply:
There appears to be no serialized block string in the `result` variable.
Do I need to generate a serialized block from the `result.attirbutes`?